File: src/codes/Users_Manual/modulation/fourier_3.cs
~using AUTD3Sharp.Modulation;
~using static AUTD3Sharp.Units;
var m = new Fourier([new Sine(100u * Hz), new Sine(150u * Hz), new Sine(200u * Hz)])
            .WithScaleFactor(1.0f).WithOffset(0).WithClamp(true);

File: src/codes/Users_Manual/modulation/fourier_3.py
~from pyautd3 import Hz, Sine
~from pyautd3.modulation import Fourier
~
m = (
    Fourier([Sine(100 * Hz), Sine(150 * Hz)])
    .with_scale_factor(1.0)
    .with_offset(0)
    .with_clamp(True)
)